azure-pipelines.yml

```
@@ -64,7 +65,39 @@ jobs:
     setup:
       - bash: |
           sudo apt-get update &&
-          sudo apt-get -y install sqlite3 libsqlite3-dev
+          sudo apt-get install wget &&
```

Use curl retry here?
```suggestion
          sudo apt-get install curl &&
```